src/cmd/ksh93/docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Is it possible to use meson instead of Makefiles ?

Reply:
Maybe but probably not worth the trouble. The makefile was autogenerated by the `sphinx-quickstart` command to set things up. If we decide to generate man files as part of `meson install` we can just do `run_command(['make', 'man'])` from a *meson.build* file.